cmd/cli/commands/miner.go

```diff
@@ -122,7 +115,7 @@ var minerStatusCmd = &cobra.Command{
 	PreRunE: checkHubAddressIsSet,
 	RunE: func(cmd *cobra.Command, args []string) error {
 		if len(args) < 1 {
-			return errMinerAddressRequired
+			return errWorkerAddressRequired
```

Comment:
if you already started renaming - how about something like 'errWorkerIdRequired' ?